exp/addons/api/v1beta1/clusterresourceset_types.go

```diff
@@ -153,7 +153,9 @@ func (m *ClusterResourceSet) SetV1Beta2Conditions(conditions []metav1.Condition)
 // +kubebuilder:storageversion
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="Time duration since creation of ClusterResourceSet"
 
-// ClusterResourceSet is the Schema for the clusterresourcesets API.
+// ClusterResourceSet is the Schema for the clusterresourcesets API
+// it provides a basic solution for installing & managing resources
+// while for advanced use cases an addon provider must be used.
```

Comment:
```suggestion
// ClusterResourceSet is the Schema for the clusterresourcesets API.
// For advanced use cases an add-on provider should be used instead.
```

The sentence in-between does not really help explaining why it is a basic solution.